js/cedar.js

```diff
@@ -69,8 +65,8 @@ Cedar.Application = function(options) {
   Cedar.config.wait = this.options.wait;
   Cedar.config.fetch = this.options.fetch;
 
-  if (typeof Cedar.$ === "undefined") {
-    Cedar.$ = jQuery({});
+  if (Cedar.events === undefined) {
```

Comment:
So I think the `typeof` thing was originally meant more for detecting things in the global state that some browsers may have no idea about.